frankenphp.c

```diff
@@ -405,7 +405,15 @@ static size_t frankenphp_ub_write(const char *str, size_t str_length)
 		return 0;
 	}
 
-	return go_ub_write(ctx->current_request ? ctx->current_request : ctx->main_request, (char *) str, str_length);
+	bool failed = false;
+
+	size_t wrote = go_ub_write(ctx->current_request ? ctx->current_request : ctx->main_request, (char *) str, str_length, &failed);
```

Comment:
You could use a multi-return function instead of passing a reference: https://stackoverflow.com/a/49957242/1352334

Reply:
TIL. Thanks!